spopp20/music_manager
# Added place-holder pages for editing and viewing an instrument or song

I added place-holder pages for editing and viewing instruments and songs.
I fixed the .env.example file adding SESSION_SECRET string that was over 32 characters long to fix the authentication stack trace.

## Setup

Edit your .env.local file and make sure your SESSION_SECRET is over 32 characters long

## Test

```
npm run dev
```

- Click Instruments and an Edit, and a View button to see the place-holder pages
- Click Songs and an Edit, and a View button to see the place-holder pages.
